tests/unit/globals/initializers.cpp

```diff
@@ -0,0 +1,15 @@
+//===---------------------------------------------------------------------===//
+//	Copyright 2022 Leaning Technologies
+//===----------------------------------------------------------------------===//
+#include <stdlib.h>
+#include <assert.h>
+
+int* p1 = (int*) malloc(900);
+int* p2 = (int*) calloc(1, 900);
+int* p3 = (int*) realloc((int*) 0, 900);
+
+int main() {
+	assert(p1);
```

Comment:
use assertEqual. It has a special implementation that works also in PreExecuter.
Also, there is a field for a message, which is nice when reading test results.
In this case you would like an assertUnequal, but you can just do assertEqual(p1!=0, true, "some msg")